add removal procedure

chilicuil opened the pull request:
Hi,

This pull request adds a clean/remove procedure (attached to prefix + Y) to remove unused plugins.

I though in Y because C and R are used in other plugins (copycat and sensible) and because U and I already are used on this plugin, so I though than YUI would be appropriated even if doesn't stand for anything related to Uninstalling/Cleaning.

bruno- commented:
Hey @chilicuil,
this is great, thanks a lot! No objections to the code at all and the tests are there - perfect :+1: We should consider adding you to the team.

Would you please just consider adding the following:

- a change to the [uninstalling plugins](https://github.com/tmux-plugins/tpm#uninstalling-plugins) and [key bindings](https://github.com/tmux-plugins/tpm#key-bindings) readme sections (keep 'em short and simple)
- addition to the [changelog file](https://github.com/tmux-plugins/tpm/blob/master/CHANGELOG.md) with your github name at the end of the line, example: `added feature <description> (@chilicuil)`

One other thing is the keybinding. It would be ideal if it was a mnemonic - so that it "stands for something" as you mentioned. All the relevant single keys seem to be taken by other plugins.
How about using `Alt` key then? `prefix + alt + u` can be a mnemonic for uninstall. It's harder to type, but that's okay as uninstall action should be more "intentional".

chilicuil commented:
Hi @bruno- ,

Thanks for the feedback =), I've updated the default keybinding to `prefix + alt + u` and added brief comments to the readme and changelog (feel free to further comment on them).

I'm not sure about the guidelines for merging on this project, so I added commits on top, if you prefer 1 commit per pull request I could smash them once the changes look ok.

bruno- commented:
Hi @chilicuil,
thanks for the work on this and sorry it took me this long to review and merge!

jrhorn424 commented:
If anyone else is on a Mac and having trouble performing the default key chord `prefix + alt + u`, follow [these instructions](https://github.com/tmux-plugins/tpm/blob/master/docs/installing_plugins_via_cmd_line.md) to run the script manually.

bruno- commented:
Hey @jrhorn424, care to open a separate issue if something is not working for you?
One thing that's mac-specific is the alt key in the terminal.app. Some users need to explicitly enable it.

Can you check the screenshot below and let me know if setting that option fixes it for you?

![screen shot 2015-07-21 at 1 48 33 pm](https://cloud.githubusercontent.com/assets/1042682/8800013/49ec1ed6-2faf-11e5-8dc7-4609319bacd3.png)

jrhorn424 commented:
@bruno- Thanks! Didn't really want to make an issue, just leave a note for others. I'm using iTerm 2, and I *did* indeed need to update my preferences based on the [FAQ](https://iterm2.com/faq.html). Great tip!
